exercises/twelve-days/twelve_days_test.go

```diff
@@ -0,0 +1,53 @@
+package twelve
+
+import (
+	"fmt"
+	"testing"
+)
+
+const targetTestVersion = 1
+
+type testCase struct {
+	input    int
+	expected string
+}
+
+var singTestCases = []testCase{
```

Comment:
We only have one set of test cases, so the variable can be `testCases` rather than `singTestCases`.